ui/overviewControls.js

```diff
@@ -64,13 +64,13 @@ class EndlessControlsManagerLayout extends OverviewControls.ControlsManagerLayou
                 break;
             case OverviewControls.ControlsState.WINDOW_PICKER:
                 workspaceBox.set_origin(0,
-                    startY + searchHeight + spacing +
-                    thumbnailsHeight + spacing * expandFraction);
+                    startY + Math.max(searchHeight + spacing,
```

Comment:
Why do we need to include the search height in this calculation at all, given that we don't display the search box in this state?

Reply:
Yeah, the reason that ended up there is that's how it was done for 41 and the end result _looks_ right. But indeed, it's weird. I gave it a closer look and I think it relates to the size allocation for the workspace thumbnails at the top, which are (with eos-desktop-extension) positioned in the same place as the search box. You can compare how space for the workspace thumbnails is allocated in the shell by defalt:

https://gitlab.gnome.org/GNOME/gnome-shell/-/blob/gnome-43/js/ui/overviewControls.js?ref_type=heads#L181-193

… and in eos-desktop-extension:

https://github.com/endlessm/eos-desktop-extension/blob/master/ui/overviewControls.js#L184-L197

I added a comment to explain it :)